extensions/common/sql/sql-core/src/main/java/org/eclipse/edc/sql/translation/FieldTranslator.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
seems an update to intelliJ's copyright profile is needed :)

Reply:
ouch, I thought I updated it but likely I did it in the wrong place